### PR TITLE
refactor: migrate to @sapphire/utilities

### PR DESCRIPTION
**Pull request by favna**
- refactor(*): migrate to @sapphire/utilities
- style(*): formatting
- refactor(skyraclient): replace last @klasa/utils reference

### CODE REVIEW

**Review by kyranet — COMMENTED** [review]
Coverage needs to be fixed.

File: package.json [hunk 1]
```diff
@@ -44,8 +44,8 @@
 		"url": "https://github.com/skyra-project/skyra/issues"
 	},
 	"dependencies": {
-		"@influxdata/influxdb-client": "^1.6.0",
 		"@influxdata/influxdb-client-apis": "^1.6.0",
+		"@influxdata/influxdb-client": "^1.6.0",
```
Comment (Quantumlyy) [inline package.json:48]: Why was this moved?

Comment (kyranet) [reply package.json:48]: Because the sort utility we use from VSCode detects `"` as a character of a higher code than `-`, so it puts the entry below, while yarn sorts by the names (which would place this one above).

That's my guess.

Comment (favna) [reply package.json:48]: yeah that indeed. It's just VSC's standard "sort alphabetically". After a while yarn will probably shift it back but I cbb to do it manually.

**Comment by Quantumlyy** [comment]
> Coverage needs to be fixed.

Couldn't we maybe do tests for something else to fix coverage that way?

**Review by kyranet — APPROVED** [review]
